templates/app-ts/services/root.ts

```diff
@@ -0,0 +1,17 @@
+import { FastifyInstance, FastifyRequest, FastifyReply } from "fastify"
+
+module.exports = function (fastify: FastifyInstance, opts: any, next: Function) {
+  fastify.get('/', function (request: FastifyRequest, reply: FastifyReply<any>) {
+    reply.send({ root: true })
+  })
+
+  next()
+}
```

Comment:
I would use the same structure of root.ts

Reply:
I guess you mean root.js :sweat_smile: 
Sure i'll do it